tutorials/speech2srt/speech2srt.py

```diff
@@ -68,10 +63,7 @@ def break_sentences(args, subs, alternative):
     for w in alternative.words:
         if firstword:
             # first word in sentence, record start time
-            start_hhmmss = time.strftime('%H:%M:%S', time.gmtime(
-                w.start_time.seconds))
-            start_ms = int(w.start_time.microseconds / 1000)
-            start = start_hhmmss + "," + str(start_ms)
```

Comment:
@lepistom the SRT library and the speech proto  both support `timedelta` there is no need to convert to string and parse back.